app/managers.py

```diff
@@ -0,0 +1,59 @@
+import sqlite3
+
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect(
+            "/MateProjectsPython/py-actor-manager/cinema.sqlite"

```

Comment:
using absolute path is bad practice, as another developer will have a different path, 
Use `OS` to specify absolute path